### PR TITLE
456 eav test suite

### PR DESCRIPTION
**Pull request by AshantiMutinta**
Implemented test suite for EAV

### CODE REVIEW

File: cas_implementations/src/eav/file/mod.rs [hunk 1]
```diff
@@ -159,171 +159,28 @@ pub mod tests {
...
-            // open
-            (None, None, None),
-        ] {
-            println!("fetch");
-            assert_eq!(
-                expected,
-                eav_storage.fetch_eav(e, a, v).expect("could not fetch eav"),
-            );
-        }
+        EAVTestSuite::test_round_trip_test(
```
Comment (ddd-mtl) [inline cas_implementations/src/eav/file/mod.rs]: Pascal case acronyms please `EavTestSuite`

Comment (AshantiMutinta) [reply cas_implementations/src/eav/file/mod.rs]: Changing asap.

File: core_types/src/cas/storage.rs [hunk 2]
```diff
@@ -231,6 +235,180 @@ where
     }
 }
 
+pub struct EAVTestSuite;
+
+impl EAVTestSuite {
+    pub fn test_round_trip_test(
```
Comment (lucksus) [inline core_types/src/cas/storage.rs]: `test_rount_trip_test` sounds like we are testing the round_trip_test.. ;)

Comment (thedavidmeister) [reply core_types/src/cas/storage.rs]: @lucksus yeah so i've been informally following a convention where `test_*` means "generate a test example thing" and `*_test` "a test for a thing"

**Pull request merged**